src/screens/MapScreen.tsx

```diff
@@ -1,16 +1,24 @@
-import React, { useState } from "react";
+import React, { Dispatch, SetStateAction, useState } from "react";
 import { Dimensions, View, StyleSheet } from "react-native";
 import MapView, { Marker } from "react-native-maps";
+import PlaceList from "../components/Places/PlaceList";
 import SelectionBox from "../components/SelectionBox";
 import { useLocation } from "../hooks/useLocation";
-import PlaceList from "../components/PlaceList/PlaceList";
 import StrollButton from "../components/StrollButton";
 
 export interface MapScreenProps {
-	toggleIsStrolling: () => void;
+	setPageStrolling: (screen: string) => void;
```

Comment:
this type is wrong i think?
Above, you're passing this in:
`setPageStrolling={goToStrollScreen}`
and above that:
`const goToStrollScreen = () => setPage("strollScreen");`
so `setPageStrolling` shouldn't have the `screen: string` argument, because you used `()` in the definition (meaning it's a function with no arguments)